server/mscalendar/views/calendar.go

```diff
@@ -64,3 +64,22 @@ func groupEventsByDate(events []*remote.Event) [][]*remote.Event {
 	}
 	return result
 }
+
+func RenderScheduleItem(s remote.ScheduleItem, timezone string) (string, error) {
+	message := "You have an upcoming event:"
+	start := s.Start.In(timezone).Time()
+	end := s.End.In(timezone).Time()
+
+	message += fmt.Sprintf("\n%s-%s (%s)", start.Format(time.Kitchen), end.Format(time.Kitchen), timezone)
```

Comment:
I'm curious what you think of using the lowercase `pm` instead of `time.Kitchen`, so it would be `3:04pm` instead of `3:04PM`. This comment made me think about this: https://github.com/mkraft/mattermost-plugin-whentochat/pull/1#issuecomment-609034663

I also think it makes sense to let the user choose the time format they want their event times to show up as. I think we agreed that on this release we assume `AM/PM` format is fine.

Reply:
That could be a wonderful help wanted ticket.